### PR TITLE
feat(tile-converter): 3d-tiles-converter progress

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts [hunk 1]
```diff
@@ -100,6 +104,15 @@ export default class Tiles3DConverter {
 
     this.slpkFilesystem = await openSLPK(inputUrl);
 
+    const preprocessResult =
+      this.slpkFilesystem || analyze ? await this.preprocessConversion() : true;
+
+    if (!preprocessResult || analyze) {
+      return;
+    }
+
```
Comment (ibgreen) [inline modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts:113]: Nit: I find the logic with repeated `||` expressions a bit hard to follow. Maybe
```suggestion
    let preprocessResult = true;
    if (analyze && this.slpkFilesystem) { 
      preprocessResult = await this.preprocessConversion();
      if (!preprocessResult) {
        return
      }
    }
```
    

    if (!preprocessResult || analyze) {
      return;
    }

```

File: modules/tile-converter/src/3d-tiles-converter/helpers/load-i3s.ts [hunk 2]
```diff
@@ -129,3 +129,23 @@ function getSlpkUrlParts(url: string): SLPKUrlParts | null {
   }
   return result;
 }
+
+/**
+ * Get nodes count inside SLPK
+ * @param fileSystem - file system of SLPK
+ * @returns number of nodes
+ */
+export async function getNodesCount(fileSystem: ZipFileSystem | null): Promise<number> {
```
Comment (ibgreen) [inline modules/tile-converter/src/3d-tiles-converter/helpers/load-i3s.ts:138]: Nit: `getNodeCount`? `countAllNodesInSLPK`?

**Review by ibgreen — APPROVED** [review]

File: modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts (hunk 1 above)
Comment (belom88) [reply modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts]: Done

File: modules/tile-converter/src/3d-tiles-converter/helpers/load-i3s.ts (hunk 2 above)
Comment (belom88) [reply modules/tile-converter/src/3d-tiles-converter/helpers/load-i3s.ts]: Done
